api/src/umigrate/settings.py

```diff
@@ -340,3 +341,12 @@
 
 MEDIA_URL = "/media/"
 MEDIA_ROOT = os.path.join(BASE_DIR, "../media")
+
+SCHEDULER_QUEUE = "{}_default".format(STAGE_ENVIRONMENT)
+RQ_QUEUES = {
+    SCHEDULER_QUEUE: {
+        "HOST": "localhost",
```

Comment:
To be consistent with other settings, let's use "127.0.0.1" instead of "localhost".